.github/workflows/codeql-analysis.yml

```diff
@@ -17,6 +17,9 @@ jobs:
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
+    - uses: actions/setup-go@v3
+      with:
+        go-version: 1.17
```

Comment:
This duplicates the `actions/setup-go@v3` step below?

Reply:
Yup. I'd guess it was a bad merge on my part (added both by me in my branch and then later in main).

I created https://github.com/cosmos/cosmos-sdk/pull/11420 to remove one of the entries.